res/database/Default/database_achievements.xml

```diff
@@ -182,30 +200,33 @@
 					<!-- kein Titel und es wird der Belohnungsinterne Text genommen -->
 					<title />
 					<text />
-					<data type="money" money="250000" />
+					<data type="money" money="100000" />
 				</reward>
 			</rewards>
 			<!-- flags = 0. "0" fuer "keine Besonderheiten" -->
 			<data flags="0" group="2" category="2" index="3" sprite_finished="gfx_datasheet_achievement_img_level3" sprite_unfinished="gfx_datasheet_achievement_img_level0" />
 		</achievement>
 		<achievement id="tvt-gameachievement-reachBroadcastArea1" creator="5578" created_by="Ronny">
 			<title>
-				<de>Sendegebiet „Bronze“</de>
+				<de>Reichweite „Bronze“</de>
```

Comment:
unintended change
After the last release a major refactoring of the map logic has been started. The translation has to be checked.